internal/metrics/client.go

```diff
@@ -94,18 +115,28 @@ func (c *client) Measure(b ginkgo.Benchmarker, f queryFunc, name string, job str
 }
 
 func (c *client) ContainerUserCPU(caJob string, duration model.Duration) (float64, error) {
+	param := "job"
+	if c.isPromOCP {
+		param = "pod"
+	}
```

Comment:
This should not be handled by passing a flag. I suggest to make the label configuration part of the `metrics` config section.